src/components/Footer.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+
+const Footer = () => (
+  <footer>
+<p  
+id='footer'
```

Comment:
Can you explain the usage for this id?

Reply:
None. My bad. typed it on automatic. Is now removed